src/bindings/html/dom.js

```diff
@@ -81,19 +81,34 @@ export function translateMutations(view, langs, mutations) {
             targets.add(addedNode);
           }
         }
+        break;
     }
   }
 
-  targets.forEach(
-    target => target.childElementCount ?
-      translateFragment(view, langs, target) :
-      translateElement(view, langs, target));
+  if (targets.size === 0) {
+    return;
+  }
+
+  let elements = [];
+
+  targets.forEach(target => {
```

Comment:
ugh, sure. But I hope you'll get familiar with multi-statement fat arrow functions one day! We can't turn them all into lambdas.